jaxrs/resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/LocalResteasyProviderFactory.java

```diff
@@ -0,0 +1,84 @@
+package org.jboss.resteasy.client.jaxrs.internal;
```

Comment:
Is it possible to also override getRuntimeType() method to return  RuntimeType.CLIENT instead of RuntimeType.SERVER? 
On ClientConfiguration this method is OK and return RuntimeType.CLIENT. But when registering any provider on it, it is the underlying ResteasyProviderFactory it wrap that is used. So for any client Feature or DynamicFeature when calling FeatureContext.getConfiguration().getRuntimeType() it will always return RuntimeType.SERVER.
Maybe I should open a separate issue for this ?